src/descriptors/analysis.rs

```diff
@@ -61,6 +62,7 @@ impl std::fmt::Display for LianaPolicyError {
             ),
             Self::InvalidPolicy(e) => write!(f, "Invalid Miniscript policy: {}", e),
             Self::PolicyAnalysis(e) => write!(f, "Analyzing the policy of the miniscript: {}", e),
+            LianaPolicyError::HardenedMultipath => write!(f, "Hardened multipaths are forbiden"),
```

Comment:
You could use `Self::` here for consistency with other lines.

Typo: "forbidden".